src/Bicep.Core.Samples/Files/InvalidModules_LF/main.syntax.bicep

```diff
@@ -348,9 +348,8 @@ module './main.bicep' = if () {
 //@[022:00023) | ├─Token(Assignment) |=|
 //@[024:00034) | └─IfConditionSyntax
 //@[024:00026) | | ├─Token(Identifier) |if|
-//@[027:00029) | | ├─ParenthesizedExpressionSyntax
+//@[027:00029) | | ├─SkippedTriviaSyntax
 //@[027:00028) | | | ├─Token(LeftParen) |(|
-//@[028:00028) | | | ├─SkippedTriviaSyntax
 //@[028:00029) | | | └─Token(RightParen) |)|
```

Comment:
This change seems related to the BCP009 diagnostics being dropped.

Reply:
The issue was here - I wasn't asserting that the inner syntax only contained one expression: https://github.com/Azure/bicep/blob/37234658dd54455a23ceb1bc8e6e1b9b787d4667/src/Bicep.Core/Parsing/Parser.cs#L637-L642

The fix was to do this:  https://github.com/Azure/bicep/blob/9bc2aa1404e17756849e36f41e50e9b35773ef17/src/Bicep.Core/Parsing/Parser.cs#L649-L654